src/functions_topic.py

```diff
@@ -0,0 +1,42 @@
+def concatenate_strings(string1: str, string2: str) -> str:
+    """
+    Returns the concatenation of two strings.
+    E.g.
+        concatenate_strings("hello", "world") -> "helloworld"
+
+    @rtype: str
+    @param string1: First string of the concatenation
+    @param string2: Second string of the concatenation
+    @return: The concatenation of string1 and string2
+    """
+    concatenated = string1 + string2
+    return concatenated
+
+
+# Tests for the concatenate_strings method
+assert "" == concatenate_strings("", "")
+assert "ab" == concatenate_strings("a", "b")
```

Comment:
Just a note that only the first failure will be shown with tests like this. Thats probably fine for learning purposes, but definitely a limitation to be aware of.

Reply:
Good point. 🤔